cmd/channel_accounts.go

```diff
@@ -62,7 +63,11 @@ func (c *ChannelAccountsCommand) Command() *cobra.Command {
 			}
 
 			// Inject server dependencies
-			svcOpts.DatabaseDSN = globalOptions.DatabaseURL
+			tssDatabaseDSN, err := router.GetDNSForTSS(globalOptions.DatabaseURL)
+			if err != nil {
+				log.Ctx(ctx).Fatalf("Error getting TSS database DSN: %v", err)
+			}
+			svcOpts.DatabaseDSN = tssDatabaseDSN
```

Comment:
Here I change the `channel-accounts` CLI command by updating the database DNS with the TSS schema (`search_path=tss`).